central/graphql/resolvers/gen/main.go

```diff
@@ -20,14 +20,17 @@ var (
 			reflect.TypeOf((*storage.ActiveComponent_ActiveContext)(nil)),
 			reflect.TypeOf((*storage.Alert)(nil)),
 			reflect.TypeOf((*storage.Cluster)(nil)),
+			//reflect.TypeOf((*storage.ClusterCVE)(nil)), TODO rox-11259
 			reflect.TypeOf((*storage.ComplianceAggregation_Response)(nil)),
 			reflect.TypeOf((*storage.ComplianceControlResult)(nil)),
 			reflect.TypeOf((*storage.CVE)(nil)),
+			reflect.TypeOf((*storage.CVEInfo)(nil)),
```

Comment:
You do not need this. Since you are generating `ImageCVE` in which `CVEInfo` is embedded, a separate resolver is generated for `CVEInfo`